lib/client/identityfile/identity.go

```diff
@@ -186,6 +215,44 @@ func writeWithNewline(w io.Writer, data []byte) error {
 	return nil
 }
 
+func writeFile(path string, data []byte, forceOverwrite bool) error {
+	if err := checkOverwrite(path, forceOverwrite); err != nil {
```

Comment:
With this approach, if the command produces 3 files (as most of them do), the user may be prompted 3 times. Think it makes sense to tweak this a bit to only ask for confirmation once if at least one of the destination files exists? The reasoning being, it's unlikely that user would want to overwrite one of the 3 and keep others intact for example.